tests/js/spec/views/alerts/incidentRules/metricField.spec.jsx

```diff
@@ -1,93 +1,101 @@
-import {mountWithTheme} from 'sentry-test/enzyme';
 import {initializeOrg} from 'sentry-test/initializeOrg';
-import {openMenu, selectByLabel} from 'sentry-test/select-new';
+import {fireEvent, mountWithTheme, screen} from 'sentry-test/reactTestingLibrary';
 
 import MetricField from 'app/views/alerts/incidentRules/metricField';
 import {Dataset} from 'app/views/alerts/incidentRules/types';
 import Form from 'app/views/settings/components/forms/form';
+import FormModel from 'app/views/settings/components/forms/model';
+
+function openSelectMenu(text) {
```

Comment:
Should we have a helper library for RTL + react-select elements like we do for enzyme and react-select?

Reply:
yes please